samples/durable-task-sdks/java/README.md

```diff
@@ -0,0 +1,94 @@
+# Java Samples for Durable Task SDK
+
+This directory contains sample applications demonstrating various patterns using the Durable Task Java SDK.
+
+## Prerequisites
+
+- Java 8 or later
+- [Docker](https://www.docker.com/get-started)
+
+## Running the samples with the Durable Task Scheduler Emulator
+
+The samples can be run locally using the Durable Task Scheduler Emulator. The emulator is a containerized version of the Durable Task Scheduler service that persists state in memory. It is useful for development and testing purposes.
+
+1. Start the Durable Task Scheduler Emulator:
+
+   ```bash
+   # Pull the emulator image
+   docker pull mcr.microsoft.com/dts/dts-emulator:v0.0.5
+
+   # Run the emulator
+   docker run -p 8080:8080 -p 8082:8082 mcr.microsoft.com/dts/dts-emulator:v0.0.5
+   ```
+
+2. Set the connection string environment variable:
+   ```bash
+   # Windows
+   set DURABLE_TASK_CONNECTION_STRING=Endpoint=http://localhost:<port>;TaskHub=default;Authentication=None
```

Comment:
Replace `<port>` with `8080` here and below since you already hardcoded the port in the `docker` commands above.